src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -1,17 +1,4 @@
 <h1>Home</h1>
-<div class="card no-border bg-light mb-4">
-  <div class="card-body">
-    <div class="form-row">
-      <div class="input-group">
-        <input id="search-input" type="text" class="form-control" [(ngModel)]="studentSearchkey" (keyup.enter)="search()"
-               placeholder="e.g. Charles Shultz, charles@gmail.com">
-        <div class="input-group-append">
-          <button id="btn-search" class="btn btn-primary" (click)="search()">Search</button>
-        </div>
```

Comment:
Is there a good reason to remove the home search bar?

Understand it's just a redirection to the search page. From the UX perspective, this removal makes search operation become two-steps-away from login.

If search is a frequent action performed by the instructor, it will be convenient to make it immediately available at home page?

Reply:
A search bar at home page heavily implies a site-wide search, which this search bar is nothing about. There are multiple cases of such mis-usage, like "how do I publish a session", "how do I enroll students", etc.

Reply:
Agree that the search bar was a bit out of place on the homepage, esp. since we already have a search page. However, I still think that the current search bar is not explicit enough as to what is searchable and what is not. I believe we can address that in a new issue or after the search migration.

Reply:
<img width="1243" alt="Screenshot 2021-03-26 at 6 45 30 PM" src="https://user-images.githubusercontent.com/7261051/112620741-e46a2480-8e63-11eb-81d4-371952492290.png">